test/test_jit_py3.py

```diff
@@ -533,6 +533,46 @@ def forward(self, x):
         # Check that ignored method is still intact.
         self.assertEqual(inp, n.ignored_method(inp))
 
+    def test_doubly_referenced_submodules(self):
```

Comment:
Should we test duplicate references to modules in other container types or even by themselves (i.e. not in a tuple or list)?